Scripts/script-CommonServer_CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## [Unreleased]
+
+
+## [19.8.0] - 2019-08-06
```

Comment:
I think it should be in the Unrealeased section

Reply:
you're right 👍